demisto_sdk/commands/lint/helpers.py

```diff
@@ -539,3 +541,17 @@ def generate_coverage_report(html=False, xml=False, report=True, cov_dir='covera
         except coverage.misc.CoverageException as warning:
             logger.warning(str(warning))
             return
+
+
+def get_linters_on_local_os(python_version: float):
+    result = DEFAULT_LINTERS_IN_OS
+    if python_version < 3:
+        result.append("mypy")
+    return result
+
+
+def get_linters_on_docker(python_version: float):
+    result = DEFAULT_LINTERS_IN_DOCKER
+    if python_version < 3:
+        result.remove("mypy")
```

Comment:
aren't we running mypy on python2 currently is this a limitation of the new version?

Reply:
limitation from the fact that Mypy run only in python 3 so python 2 images doesn't have python 3 and can't run Mypy 
in that case Mypy will run locally